rust/noosphere-ns/src/lib.rs

```diff
@@ -1,5 +1,24 @@
+#[cfg(not(target_arch = "wasm32"))]
 #[macro_use]
 extern crate tracing;
 
 #[cfg(not(target_arch = "wasm32"))]
 pub mod dht;
+
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
Is there a way to group these `cfg` conditionals into a mod or something? Hella noisy

Reply:
Beautiful